portal/app/home_updater/updater.py

```diff
@@ -0,0 +1,20 @@
+import logging
+import app.home_updater.std_api as std_api
+import app.home_updater.sonnen_api as sonnen_api
+import app.home_updater.egauge_api as egauge_api
+from app.algorithms import battery_load_control, battery_optimizer
+from apscheduler.schedulers.background import BackgroundScheduler
+
+
+logging.basicConfig()
+
+
+def start():
+    scheduler = BackgroundScheduler()
+
+    scheduler.add_job(sonnen_api.update_battery_status, 'interval', minutes=5)
+    scheduler.add_job(std_api.update_std_device_status, 'interval', minutes=5)
+    scheduler.add_job(egauge_api.update_egauge_data, 'interval', seconds=5)
+    # scheduler.add_job(battery_optimizer.batt_opt, 'interval', minutes=5)
+    # scheduler.add_job(battery_load_control.batt_dispatch, 'interval', minutes=5)
+    scheduler.start()
```

Comment:
Im not sure if we are going to have an issue by starting multiple schedulers, one from home_updater and another from farm_updater. We might need to consolidate them to a single scheduler with multiple jobs. Have you tried running locally having two schedulers?

Reply:
Yes it works locally. But I agree it would be better to organize them into one updater if we always expect them to be scheduled together. Maybe combine app/farm_api.py and app/home_device_api.py into one file and move both updaters into it?

Reply:
yes, that's what I would suggest!

Reply:
Please check the latest commit.